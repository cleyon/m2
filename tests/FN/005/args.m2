@define intro Here are >$*< !!
Foo @intro Flopsy, Mopsy, Cotton-tail@
