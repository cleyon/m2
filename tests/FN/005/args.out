Foo Here are >Flopsy, Mopsy, Cotton-tail< !!
